packages/core/src/Page/index.js

```diff
@@ -796,6 +796,14 @@ class Page {
     }
   }
 
+  adjustHighlightStyle() {
+    const $ = cheerio.load(this.content);
```

Comment:
hmm, parsing the entire document for this seems pretty expensive (its current liberal use is also our main performance bottleneck); let's try to to move it to the main html processing area in `ComponentParser`